opus/application/static_media/js/browse.js

```diff
@@ -51,18 +51,18 @@ var o_browse = {
         $(".gallery-contents, .dataTable").on('scroll', _.debounce(o_browse.checkScroll, 500));
 
         $(".gallery-contents, .dataTable").on('wheel ps-scroll-up', function(event) {
-            let namespace = o_browse.getViewInfo().namespace;
-            // if time to load...
-                if (opus.prefs.browse === "dataTable") {
-                    if ($(`${namespace} .dataTable`).scrollTop() === 0) {
-                        $(`${namespace} .gallery-contents`).infiniteScroll("loadNextPage");
-                    }
-                } else {
-                    if ($(`${namespace} .gallery-contents`).scrollTop() === 0) {
-                        $(`${namespace} .gallery-contents`).infiniteScroll("loadNextPage");
-                    }
+            let view = o_browse.getViewInfo();
+            let tab = view.namespace;
+            if (opus.prefs[`${view.prefix}startobs`] > 0) {
+                let prev = $(`${view.namespace} [data-obs]`).first().data("obs") - o_browse.getLimit();
+                if ($(`${tab} .gallery-contents`).scrollTop() || $(`${tab} .dataTable`).scrollTop() === 0) {
```

Comment:
The use of || here is confusing me. As written, this says "If gallery-contents scrollTop is NOT zero, OR dataTable scrollTop IS zero". Don't you want to say if either is zero?